site2/docs/client-libraries-java.md

```diff
@@ -140,7 +140,7 @@ If you create a client, you can use the `loadConf` configuration. The following
 `numIoThreads`| int| The number of threads used for handling connections to brokers | 1 
 `numListenerThreads`|int|The number of threads used for handling message listeners. The listener thread pool is shared across all the consumers and readers using the "listener" model to get messages. For a given consumer, the listener is always invoked from the same thread to ensure ordering. If you want multiple threads to process a single topic, you need to create a [`shared`](/concepts-messaging.md#shared) subscription and multiple consumers for this subscription. This does not ensure ordering.| 1 
 `useTcpNoDelay`| boolean| Whether to use TCP no-delay flag on the connection to disable Nagle algorithm |true
-`useTls` |boolean |Whether to use TLS encryption on the connection| false
+`enableTls` |boolean | Whether to use TLS encryption on the connection. **Deprecated**. use "pulsar+ssl://" in serviceUrl to enable | false
```

Comment:
what does `Deprecated` mean? `enableTls` is deprecated or something else？

Reply:
There is no need to config `enableTls` in pulsar client to enable Tls, just use `pulsar+ssl://` in the connection address